Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -820,6 +861,79 @@ public Task Sort( DataGridColumn<TItem> column, SortDirection? sortDirection = n
         return Task.CompletedTask;
     }
 
+
+    /// <summary>
+    /// Replaces the current sorting with the given sort order

```

Comment:
Do you think this would explain it better?
"Applies a new sorting definition to the Datagrid. Replaces the current sorting."

Reply:
Yes, thanks.

Fixed in 01f883a751824450f29f02939fea1726012fcfe0